src/Functions/randDistribution.cpp

```diff
@@ -226,11 +225,20 @@ class FunctionRandomDistribution : public IFunction
                             parameter_number, arguments.size());
 
         const IColumn * col = arguments[parameter_number].column.get();
+        ResultType parameter;
```

Comment:
Shouldn't be ResultType, if argument will be Int64, and result Float64, it will cause invalid execution